StyleGuidelines.md

```diff
@@ -1285,6 +1286,38 @@ ls -File $root -Recurse | ? { @('.gitignore', '.mof') -contains $_.Extension }
 Get-ChildItem -File $root -Recurse | Where-Object { @('.gitignore', '.mof') -contains $_.Extension }
 ```
 
+### Capitalized cmdlets, Function Names and Pester Assertions
+
+PowerShell cmdlets, Functions and Pester assertions should all start with capital letters.  This makes code easier to read, especially when there are multiple nouns in the command name.  
```

Comment:
The part about capitalizing cmdlets/function names is already in the Functions section of this document.
Can you possibly edit this section to just be about the Pester assertions so that we don't duplicate guidelines?